build/ci/release.yml

```diff
@@ -575,23 +578,23 @@ tasks:
       - func: "generate download archive json"
         vars:
           FEED_FILE_NAME: mongodb-atlas-cli.json
-      - func: "upload dist"
-      - command: s3.put
-        params:
-          aws_key: ${download_center_aws_key}
-          aws_secret: ${download_center_aws_secret}
-          local_files_include_filter:
-            - src/github.com/mongodb/mongocli/dist/*.tar.gz
-            - src/github.com/mongodb/mongocli/dist/*.zip
-            - src/github.com/mongodb/mongocli/dist/*.deb
-            - src/github.com/mongodb/mongocli/dist/*.rpm
-            - src/github.com/mongodb/mongocli/dist/*.tgz
-            - src/github.com/mongodb/mongocli/dist/*.json
-          remote_file: mongocli/
-          bucket: downloads.mongodb.org
-          permissions: public-read
-          content_type: ${content_type|application/x-gzip}
-          display_name: downloads-center-
+#      - func: "upload dist" # WAITING FOR https://jira.mongodb.org/browse/WEBSITE-11811
```

Comment:
we are not adding the atlascli binaries to the download center bucker until we know if we can reuse the same folder

Reply:
[q] can we make this a TODO so we can easily scan in the future or lint doesn't allow us?

Reply:
updated